### PR TITLE
PORTALS-1916

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/lib/utils/SynapseClient.ts [hunk 1]
```diff
@@ -92,12 +92,17 @@ import { VersionInfo } from './synapseTypes/VersionInfo'
 import { SearchQuery, SearchResults } from './synapseTypes/Search'
 import { ResearchProject } from './synapseTypes/ResearchProject'
 import {
-  ManagedACTAccessRequirementStatus,
   RequestInterface,
```
Comment (emmanmills) [inline src/lib/utils/SynapseClient.ts:95]: Was there a merge conflict? This line is required for the ManagedACTAccess component.

File: src/lib/utils/SynapseClient.ts [hunk 2]
```diff
@@ -2229,7 +2234,7 @@ export const getAccessRequirement = (
 export const getAccessRequirementStatus = (
   sessionToken: string | undefined,
   requirementId: string | number,
-): Promise<AccessRequirementStatus | ManagedACTAccessRequirementStatus> => {
+): Promise<AccessRequirementStatus> => {
   const url = `repo/v1/accessRequirement/${requirementId}/status`
```
Comment (emmanmills) [inline src/lib/utils/SynapseClient.ts:2238]: Same as above. This line is required for the ManagedACTAccess component.

File: src/lib/utils/SynapseClient.ts (hunk 1 above)
Comment (jay-hodgson) [reply src/lib/utils/SynapseClient.ts]: Nice catch, this was a bad merge from conflict resolution

File: src/lib/utils/SynapseClient.ts (hunk 2 above)
Comment (jay-hodgson) [reply src/lib/utils/SynapseClient.ts]: added this back in

**Review by emmanmills — APPROVED** [review]
Looks good

**Pull request merged**